packages/application-shell/src/components/navbar/use-navbar-state-manager.ts

```diff
@@ -125,9 +127,14 @@ const useNavbarStateManager = (props: HookProps) => {
     STORAGE_KEYS.IS_FORCED_MENU_OPEN
   );
   const isForcedMenuOpen = isNil(cachedIsForcedMenuOpen)
-    ? null
+    ? isForcedMenuOpenDefaultValue
     : (JSON.parse(cachedIsForcedMenuOpen) as boolean);
 
+  window.localStorage.setItem(
+    STORAGE_KEYS.IS_FORCED_MENU_OPEN,
+    String(isForcedMenuOpenDefaultValue)
```

Comment:
Shouldn't we be using here `isForcedMenuOpen`?

If we don't and the cached value is `true`, the we would be using `true` in next code flow but we would be storing `false` in the local storage.

Reply:
Oh, right, good point 👍